cmd/rule-evaluator/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.22-bullseye AS buildbase
+FROM google-go.pkg.dev/golang:1.22.0@sha256:564af112166fe661bf3b29e2ffd687868eaa0a7b9f02d7056e3e62f9df991b89 as buildbase
```

Comment:
Question: Is there a way to list the available images in that repo? 1.22.1 should be released, I think.

Reply:
Yep - using the [`gcrane` tool](https://cloud.google.com/artifact-registry/docs/docker/copy-from-gcr#copy-gcrane), i.e. `gcrane ls google-go.pkg.dev/golang`.

Good catch, updated to 1.22.1